cvat/apps/engine/cloud_provider.py

```diff
@@ -961,9 +967,11 @@ def wrapper(self, key=None):
             else:
                 func(self, key)
             return Status.AVAILABLE
-        except GoogleCloudNotFound:
+        except GoogleCloudNotFound as e:
+            slogger.glob.warning(f"GoogleCloudNotFound: {e}")
             return Status.NOT_FOUND
-        except GoogleCloudForbidden:
+        except GoogleCloudForbidden as e:
+            slogger.glob.warning(f"GoogleCloudForbidden: {e}")
```

Comment:
I see that you didn't add any new handlers here; is the "endpoint not available" case covered by one of the existing handlers?